AppInspector.CLI/ResultsWriter.cs

```diff
@@ -58,20 +58,19 @@ public static void Write(Result result, CLICommandOptions options)
                     writer?.WriteResults(analyzeResult, cLIAnalyzeCmdOptions);
 
                     //post checks
-                    if (File.Exists(options.OutputFilePath) && new FileInfo(options.OutputFilePath).Length > MAX_HTML_REPORT_FILE_SIZE)
+                    if (options.OutputFilePath is not null && File.Exists(options.OutputFilePath) && new FileInfo(options.OutputFilePath).Length > MAX_HTML_REPORT_FILE_SIZE)
```

Comment:
This is the output file path, right? What if the file doesn't exist?  Is it created before this line?
File.Exists(options.OutputFilePath)

Reply:
Line 58 creates the file.